srcpkgs/geonkick/template

```diff
@@ -9,9 +9,10 @@ makedepends="jack-devel libsndfile-devel rapidjson
 short_desc="Free software percussion synthesizer"
 maintainer="teldra <teldra@rotce.de>"
 license="GPL-3.0-or-later"
-homepage="https://github.com/iurie-sw/geonkick"
-distfiles="https://github.com/iurie-sw/geonkick/archive/v${version}.tar.gz"
-checksum=ec656e76df13ae1226bba6eeb8146c4f0ddbae2c34d28a89db473c16497eaff4
+homepage="https://github.com/free-sm/geonkick"
+changelog="https://github.com/free-sm/geonkick/blob/main/NEWS.md"
+distfiles="https://github.com/free-sm/geonkick/archive/refs/tags/v${version}.tar.gz"
+checksum=813fb10709136ed79403c8baf3d847fce9fbf717fea3c194ad2952e0172e6d45
 make_check=extended # No target to "ninja test".
```

Comment:
This is interesting, do you remember why you added this? "No target to ninja test" is just a harmless warning, this doesn't really make sense to me.